Makefile.validation

```diff
@@ -36,8 +36,8 @@ pr-validation-junit:
 
 # Run e2e tests for CI. All of these tests use ginkgo so we can use native ginkgo methods to print junit output.
 # Also go test doesn't provide functionnality to save test's log into the file. Use > to archieve artifatcs.
-test-ci:
-	${GO_ENV_VARS} CI=true go test -v test/e2e/baremetal_e2e_test.go -ginkgo.v -ginkgo.progress -kubeconfig=${HOME}/.kube/config -timeout=0 > log.txt
+test-ci-with-operator:
```

Comment:
Why did you rename?

Reply:
It was needed for testing in CI. Renamed back